statsmodels/graphics/gofplots.py

```diff
@@ -308,10 +309,37 @@ def qqplot(self, xlabel=None, ylabel=None, line=None, other=None,
             if not check_other:
                 other = ProbPlot(other)
 
-            fig, ax = _do_plot(other.sample_quantiles,
-                               self.sample_quantiles,
-                               self.dist, ax=ax, line=line,
-                               **plotkwargs)
+            if self.nobs < other.nobs:
+                finv = interp1d([(i+1) / other.nobs for i in range(other.nobs)], other.sorted_data)
```

Comment:
This problem is completely symmetric.  I should be possible to always fix the long series and then just call a single version of _do_plot